### DISCUSSION TITLE
Installation of Optuna Dashboard

### DISCUSSION THREAD

**Discussion by sreeaurovindh** [optuna/optuna-dashboard#129] [Q&A]
After Installing optuna-dashboard , when I start I get 'command not found'. Any help on how to fix it

Thanks

**Comment by c-bata** [comment]
If you are using venv for dependency management, try following commands:

```
$ python3 -m venv venv
$ source venv/bin/activate
(venv) $ python -m pip install optuna-dashboard
(venv) $ optuna-dashboard sqlite:///db.sqlite3
```

**Comment by sreeaurovindh** [comment]
Thanks. For some reason, restarting the system fixed the issue

Regards ,
Sree

On Tue, Aug 17, 2021 at 10:09 PM Masashi Shibata ***@***.***>
wrote:

> If you are using venv for dependency management, try following commands:
>
> $ python3 -m venv venv
> $ source venv/bin/activate
> (venv) $ python -m pip install optuna-dashboard
> (venv) $ optuna-dashboard sqlite:///db.sqlite3
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/optuna/optuna-dashboard/discussions/129#discussioncomment-1198050>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA3QWREQRGKHCGJRZQMH2GTT5MI5XANCNFSM5CJZOJRQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
-- 
Thanks,
Sree Aurovindh Viswanathan
Ph.D. Candidate in Computer Science
Graduate Research Assistant
Fulton School of Computing, Informatics and Decision System Engineering
Arizona State University (Tempe)
Ph: +1-480-399-3390
Linkedin : https://www.linkedin.com/in/sreeaurovindh/
